ocaml/testsuite/tests/typing-modules/inclusion_errors_elision.ml

```diff
@@ -27,7 +27,13 @@ Lines 9-13, characters 15-3:
 12 |   end
 13 | end
 Error: Signature mismatch:
-       ...
+       Modules do not match:
+         sig
+           module B :
+             sig type a and b and c and d and e and f and g and h end
+         end
+       is not included in
+         S
```

Comment:
The tests in this file are supposed to show that ellision works as expected, and this test no longer appears to show that. Can you change it to show that?

Reply:
The original file contains two tests, both have the eliding logic triggered, and I can't see the difference between the two tests.

The current file has two tests - one triggers and one not. Maybe that's better?

Reply:
I think the test is meant to exercise a different code path: the one that always prints an `Mt_core` message even in the event of the starting message being elided. See https://github.com/ocaml/ocaml/commit/8218be9e2b24907b8558776a34d12032bcc42496#diff-c6741d931dca5b1ee5f6d02173f0370101a831c62b91f4c7c4c58f78bbf23de2. I think we should have both tests in this file exhibit elision.

Reply:
OK I've added two more tests. Now we have 2x2 tests corresponding to all cases.